rllib/examples/multi_agent/two_algorithms.py

```diff
@@ -0,0 +1,188 @@
+# TODO (sven): Move this example script into the new API stack.
```

Comment:
Here as well: Is this possible with new stack and can we otherwise put this example under a specific foler named `old_stack`?

Reply:
There is a new folder called `_old_api_stack`, but I would like to only put scripts in there that don't need to be translated to the new API stack b/c whatever is demo'd in these will no longer be supported (or relevant).

All the other scripts currently still on the old stack (that are NOT in that folder) will have to be translated.

I agree with you that we should mark all these differences more clearly.
Suggestion:
* Old-API stack scripts that will NOT be supported in the future (those in `_old_api_stack/`): add `deprecation_warning` to top explaining that these scripts will be deprecated fully (w/o replacement)
* Old-API stack scripts that we will translated (those NOT in `_old_api_stack/`): add warning to top explaining that we are about to translate these to the new stack (and will retire/remove their old stack version).
* New API stack: Leave as-is (this should be the new normal).

Reply:
Yes, it should be possible. This is not even new stack specific. We just have to use PPO/DQNRLModule instead of PPO/DQNPolicy.